client-app/src/desktop/common/SampleGridModel.js

```diff
@@ -192,7 +192,8 @@ export class SampleGridModel {
             gridModel = this.gridModel;
 
         gridModel.loadData(trades, summary);
-        if (!gridModel.hasSelection) gridModel.selectFirst();
+        // if agApi is null, grid is not ready
+        if (gridModel.agApi && !gridModel.hasSelection) gridModel.selectFirst();
     }
```

Comment:
This is admittedly a very weird bit of code to have to write to avoid the warning.  We have discussed adding a public getter of the form grid.isReady(). In any case, comment is more verbose than needed for this code...